packages/lexical-editor/src/nodes/ListNode.ts

```diff
@@ -137,25 +146,56 @@ export class WebinyListNode extends ElementNode {
         };
     }
 
-    override updateDOM(prevNode: WebinyListNode, dom: HTMLElement, config: EditorConfig): boolean {
+    /*
+     * Set default styleId from first style that is found in the theme that contains current ul or ol tag
+     */
+    setDefaultThemeListStyleByTag(tag: string, theme: WebinyTheme) {
+        if (!tag) {
+            return;
+        }
+
+        const themeEmotionMap = theme?.emotionMap;
+        if (!themeEmotionMap) {
+            return;
+        }
+
+        const style = findTypographyStyleByHtmlTag(tag, themeEmotionMap);
```

Comment:
I noticed this doesn't work for ordered list. Reported it here https://webiny.slack.com/archives/C01BL618QLA/p1685357931418519

Reply:
Functionally works in the correct way, if there is no theme style the `typography action will be removed` from the toolbar. 
We currently don't have a theme style for the ordered list. That would require migration and we will postpone for another version.